pkg/detector/ospkg/rootio/provider_test.go

```diff
@@ -35,6 +35,15 @@ func TestProvider(t *testing.T) {
 			},
 			want: true,
 		},
+		{
+			name:     "Rocky with .root.io package",
+			osFamily: ftypes.Rocky,
+			pkgs: []ftypes.Package{
+				{Name: "libc6", Version: "2.31-0rocky9.9.root.io"},
+				{Name: "bash", Version: "5.1-60rocky1"},
```

Comment:
We contaminate version from multiple fields (version + release + epoch).
Use correct package here.
example from Rocky:
https://github.com/aquasecurity/trivy/blob/3b1426a67625aff99470673e63229698d0ed3091/pkg/detector/ospkg/rocky/rocky_test.go#L42-L50